src/core/analytics/visibilityanalyticshandler.js

```diff
@@ -16,19 +22,19 @@ export default class VisibilityAnalyticsHandler {
    * Initialize visibility change event listener(s) to send analytics events
    * when a result page have become visible or have been hidden.
    */
-  initVisibilityChangeListeners () {
+  initVisibilityChangeListeners (storage) {
```

Comment:
nit: can you add the type of 'storage' to the jsdoc?

Reply:
sure thing!